src/pages/in/embed/_id/button.vue

```diff
@@ -140,12 +141,25 @@ export default {
       }
     },
     async doLike() {
-      if (!this.isMaxLike) {
+      if (!this.isMaxLike && !this.isCreator) {
         this.like();
         logTrackerEvent(this, 'LikeButtonFlow', 'clickLike', 'clickLike(embed)', 1);
       } else if (this.canSuperLike) {
         await this.newSuperLike();
         await this.updateSuperLikeStatus();
+      } else {
+        if (this.ctaClickCount === undefined) {
+          this.ctaClickCount = 1;
+        } else {
+          this.ctaClickCount = (this.ctaClickCount + 1) % 5;
+          if (this.ctaClickCount !== 0) return;
```

Comment:
Handling civic CTA in `doLike` seems strange and confusing, maybe should split cta part into separated function and call that from here instead